task_reactredux/task1/src/containers/CounterContainer.js

```diff
@@ -31,14 +32,58 @@ class CounterContainer extends React.Component {
         });
```

Comment:
Probably it's better to use arrow functions here, I mean for all functions in classes instead of 
```
     decrement() {
         this.setState({
             count: this.state.count - 1
         });
     }
```
use `decrement = () => {}`. After this, you could remove `this.decrement = this.decrement.bind(this);`.